apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -40,29 +40,6 @@ function is_block_editor_screen() {
 	return is_callable( 'get_current_screen' ) && get_current_screen() && get_current_screen()->is_block_editor();
```

Comment:
We can also remove this function as it's only used by `is_homepage_title_hidden`